app/assets/javascripts/application.js.erb

```diff
@@ -22,6 +22,7 @@
 //= require klass
 //= require algoliasearch.min
 //= require_directory ./settings/
+//= require mailcheck
```

Comment:
Could you move this require to `user/application.js`? We are using mailcheck only in that bundle, not in the whole application.